plutus-use-cases/test/Spec/Crowdfunding.hs

```diff
@@ -169,9 +169,14 @@ tests = testGroup "crowdfunding"
         "test/Spec/contractError.txt"
         (pure $ renderWalletLog (void $ Trace.activateContractWallet w1 con))
 
-    -- TODO: Linked to https://github.com/input-output-hk/plutus-apps/issues/754
-    -- Re-activate once issue is resolved
-    -- , testProperty "QuickCheck ContractModel" prop_Crowdfunding
+    , testProperty "QuickCheck ContractModel" prop_Crowdfunding
+
+    , testProperty "start-at-slot-20" $ withMaxSuccess 1 $
+        let fixedTestCase = do
+              action $ CContribute w6 (Ada.lovelaceValueOf 20000000)
+              waitUntilDL (Slot 20)
+              action CStart
+        in forAllDL fixedTestCase prop_Crowdfunding
```

Comment:
Do we want to keep these fixed test cases? They should normally be generated by `testProperty "QuickCheck ContractModel" prop_Crowdfunding`

Reply:
This one is pretty tricky, it only fails with exactly slot 20 and no wait at the end. So the chance that QC generates it is pretty small. Otherwise I'd agree.

Reply:
Good point.